.gitignore

```diff
@@ -7,7 +7,4 @@ __pycache__/
 
 # Twisted Unit Test Artifacts
 _trial_temp*/
-_trial_temp*
-
-# Don't commit keys
-resources/keys/
```

Comment:
I removed the standalone browser integration, which also means the resources/keys is not used anymore.